build/task-link.x

```diff
@@ -15,6 +15,16 @@ SECTIONS
     __etext = .;
   } > FLASH =0xdededede
 
+   /* 32-byte alignment requirement for SAU */
```

Comment:
What effect does this have on non-TZ binaries? It seems like it'll inject some padding.